<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectProperties</class>
 <widget class="QDialog" name="ProjectProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectProperties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="menuList">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>291</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Board name</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cbBoardName">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>411</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QRadioButton" name="rbNoExternalProgrammer">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>230</y>
             <width>441</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>No. Upload program via USB cable</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="rbYesExternalProgrammer">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>260</y>
             <width>441</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Yes. Use the following external programmer</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>351</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Serial port (Select any port to refresh this list)</string>
           </property>
          </widget>
          <widget class="QSerialSelector" name="cbSerialPort">
           <property name="geometry">
            <rect>
             <x>22</x>
             <y>160</y>
             <width>91</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>210</y>
             <width>411</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Use an external programmer?</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cbExternalProgrammer">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>290</y>
             <width>391</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lblName">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>General options:</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2"/>
         <widget class="QWidget" name="page_3"/>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSerialSelector</class>
   <extends>QComboBox</extends>
   <header location="global">qserialselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
